airflow/models/taskinstance.py

```diff
@@ -723,20 +723,16 @@ def current_state(self, session=NEW_SESSION) -> str:
 
         :param session: SQLAlchemy ORM Session
         """
-        ti = (
-            session.query(TaskInstance)
+        state = (
```

Comment:
```suggestion
        return (
```

I think this would work, and is even simpler? (Sorry, has to be across 3 suggestions 🤷‍♂️)